patroni/request.py

```diff
@@ -136,6 +136,21 @@ def reload_config(self, config: Union[Config, Dict[str, Any]]) -> None:
         cacert = self._get_ctl_value(config, 'cacert') or self._get_restapi_value(config, 'cafile')
         self._apply_pool_param('ca_certs', cacert)
 
+    @staticmethod
+    def get_restapi_url(base: Optional[str], endpoint: Optional[str]) -> str:
```

Comment:
It would be more logical to make it a method of the Member class.
For example, `get_endpoint_url(self, endpoint: Optional[str] = None) -> str`